api/types/fuzz_test.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This file is missing `go1.18` build tag.

@zmb3 Do we have any plans to enable Go 1.18 in our `go.mod`? Looks like we're using 1.18 in many places now and adding the tags to every new fuzz test file doesn't sound very efficient.

Reply:
I actually recommended removing the 1.18 build tags, because I don't want fuzzing to stop working when we upgrade to Go 1.19, and master is already on 1.18.

I'm fine with changing go.mod to say 1.18 if that's required in order to use fuzzing.

Reply:
I removed them from all fuzz test files.